MoviesApp/Controller/BaseTableViewController.swift

```diff
@@ -74,5 +91,9 @@ extension BaseTableViewController: MovieCellDelegate {
         reloadFilteredMovies()
         tableView.reloadRows(at: [indexPath], with: .none)
     }
-    
+}
+
+extension Notification.Name {
+    static let datasourceChanged = Notification.Name("datasourceChanged")
```

Comment:
These names should be capitalized, so that we use the like:
Notification.Name.NSCalendarDayChanged